modules/ossm-rn-known-issues.adoc

```diff
@@ -15,30 +15,44 @@ Module included in the following assemblies:
 
 These limitations exist in {SMProductName}:
 
-* {SMProductName} does not yet support link:https://issues.redhat.com/browse/MAISTRA-1314[IPv6], as it is not yet fully supported by the upstream Istio project.  As a result, {SMProductName} does not support dual-stack clusters.
+* {SMProductName} does not yet fully support link:https://issues.redhat.com/browse/MAISTRA-1314[IPv6]. As a result, {SMProductName} does not support dual-stack clusters.
```

Comment:
Does not? Don't we want to announce support for IPv6 except federation deployments?

Reply:
I discussed this with Jamie and this entry was removed from the rel notes. The source had a comment to retain this until IPv6 was supported, so it can now be removed.